worker-batch-ingestion-plugin/src/main/java/com/github/cafdataprocessing/worker/ingestion/IngestionBatchWorkerPlugin.java

```diff
@@ -192,7 +191,16 @@ private void handleSubbatch(final String subbatch, final BatchWorkerServices bat
             batchWorkerServices.registerItemSubtask(DocumentWorkerConstants.DOCUMENT_TASK_NAME, 1, document);
         }
     }
-
+    
+    private void throwBatchExceptionIfFileDoesNotExist(final String subbatch, final Path subbatchFileName) throws BatchDefinitionException
```

Comment:
- Utility methods should be `static`.
- Consider changing the name to `throwBatchExceptionIfFileIsNotAccessible`.  I think that may be more accurate.